pkgs/applications/networking/instant-messengers/zoom-us/default.nix

```diff
@@ -81,7 +81,7 @@ in mkDerivation {
     # Desktop File
     cp ${desktopIntegration}/us.zoom.Zoom.desktop $out/share/applications
     substituteInPlace $out/share/applications/us.zoom.Zoom.desktop \
-        --replace "Exec=zoom" "Exec=$out/bin/zoom-us"
+        --replace "Exec=zoom" "Exec=$out/share/zoom-us/ZoomLauncher"
```

Comment:
Why does this have to change if you still provide `/bin/zoom-us`?

Reply:
actually, you're right here.